mercure.rst

```diff
@@ -0,0 +1,564 @@
+.. index::
+   single: Mercure
```

Comment:
If this is a component shouldn’t it be moved in components/mercury.rst ?

Reply:
This a components + a bundle. Here it’s the broad overview of how to push with Symfony, and most features described require the bundle and the recipe.

I’ll also add a file in components/ as done for Messenger, Validator, WebLink...

Reply:
Maybe splitting the basics which are not Symfony related to the components rst file?

Reply:
I don’t like this approach because it hurts the learning experience. I’ll
duplicate the basics like in
https://symfony.com/doc/current/components/web_link.html.

Reply:
@OskarStark I added a tiny doc entry for the component itself.